src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,52 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+  };
+
+  static addAnimal(animal) {
+    this.alive.push(animal);
+  }
+
+  static removeAnimal(animal) {
+    const index = this.alive.findIndex(el => el === animal);
```

Comment:
specify what index are you looking for, when your class will grow up it will be easier to read, for example killedAnimalIndex, also `el` - is not good name for variable

Reply:
renamed